documentation/sphinx/source/release-notes/release-notes-620.rst

```diff
@@ -8,6 +8,8 @@ Release Notes
 * Fix backup agent stall when writing to local filesystem with slow metadata operations. `(PR #4428) <https://github.com/apple/foundationdb/pull/4428>`_
 * Backup agent no longer uses 4k block caching layer on local output files so that write operations are larger. `(PR #4428) <https://github.com/apple/foundationdb/pull/4428>`_
 * Fix accounting error that could cause commits to incorrectly fail with ``proxy_memory_limit_exceeded``. `(PR #4529) <https://github.com/apple/foundationdb/pull/4529>`_
+* Added support for downgrades from FDB version 6.3. `(PR #4673) <https://github.com/apple/foundationdb/pull/4673>`_
+* Restrictions added for 6.3 clusters to maintain compatibility with a 6.2 downgrade. Details available in ``administration.rst``. `(PR #4469) <https://github.com/apple/foundationdb/pull/4469>`_
```

Comment:
My suggestion would be to combine these into one release note that says something like:

* Added support for downgrades from FDB version 6.3. For more details, see :ref:`<link to downgrade section in admin guide>`.